src/osd/OSD.h

```diff
@@ -2082,12 +2082,12 @@ class OSD : public Dispatcher,
   void handle_pg_stats_ack(class MPGStatsAck *ack);
   void flush_pg_stats();
 
-  ceph::coarse_mono_clock::time_point last_sent_beacon;
+  utime_t last_sent_beacon;
```

Comment:
This isn't going to work, we're trying to move to the standards-provided time structures. Why do you want it the other way around?